src/components/ground-nav/ground-nav.scss

```diff
@@ -2,215 +2,256 @@
+ * 1. Bottom-align to align with last menu items
+ * 2. Reposition in top-right corner
  */
-.c-ground-nav__menu-items {
-  column-gap: $_ground-nav-space;
-  columns: 2;
+.c-ground-nav__social {
+  align-self: end; // 1
+  grid-area: social;
+
+  @media (width >= breakpoint.$xl) {
+    align-self: start; // 2
+    justify-self: end; // 2
+  }
 }
 
-.c-ground-nav__kudos,
-.c-ground-nav__legal {
-  font-size: size.$font-small;
+.c-ground-nav__social-icon {
+  --icon-size: #{size.$icon-medium};
+
+  @media (width >= breakpoint.$xl) {
+    --icon-size: #{size.$icon-large};
+  }
 }
 
 /**
- * We use CSS grid when supported by the browser.
+ * 1. Bottom-align to align with last menu items
+ * 2. Gives a little breathing room to the illustration
  */
-@supports (display: grid) {
-  .c-ground-nav__content-inner {
-    display: grid;
-    grid-gap: $_ground-nav-space;
-    margin: 0 auto;
-
-    /**
-     * As the screen size increases, the number of columns increases.
-     * Instead of making all the columns the same size, we use `minmax`
-     * for one of the columns to make sure the content in this section
-     * doesn't have any line breaks, but can still grow as horizontal
-     * space allows.
-     */
-    @media (min-width: breakpoint.$m) {
-      grid-template-columns: 1fr minmax(max-content, 1fr);
-    }
-
-    @media (min-width: breakpoint.$l) {
-      grid-template-columns: 1fr minmax(max-content, 1fr) 1fr;
-    }
-  }
-
-  /**
-   * Removes the margin used by our fallback design. Grid allows us to
-   * use `grid-gap` to create consistent space between elements.
-   */
-  .c-ground-nav__content-inner > * + * {
-    margin: 0;
-  }
-
-  /**
-   * The visual order of the footer content is different than the
-   * semantic order. We explicitly define where content should
-   * be placed within the grid.
-   */
-  .c-ground-nav__organization {
-    @media (min-width: breakpoint.$m) {
-      grid-column: 1/2;
-      grid-row: 1/3;
-    }
-  }
-
-  .c-ground-nav__menu {
-    @media (min-width: breakpoint.$m) {
-      border: none;
-      grid-column: 2/3;
-      grid-row: 1/3;
-      padding: 0;
-    }
-  }
-
-  .c-ground-nav__social {
-    border: 0 $_ground-nav-border-style $_ground-nav-border-color-fallback;
-    border-color: $_ground-nav-border-color;
-
-    @media (min-width: breakpoint.$m) {
-      border-block-end-width: $_ground-nav-border-width;
-      grid-column: 1/3;
-      padding-block-end: $_ground-nav-space;
-    }
-
-    @media (min-width: breakpoint.$l) {
-      border-block-end-width: 0;
-      grid-column: 3/4;
-      grid-row: 1/2;
-      justify-self: end;
-      padding-block-end: 0;
-    }
-  }
-
-  .c-ground-nav__kudos {
-    @media (min-width: breakpoint.$m) {
-      grid-column: 1/3;
-    }
-  }
-
-  .c-ground-nav__legal {
-    @media (min-width: breakpoint.$l) {
-      align-self: end;
-      justify-self: end;
-    }
+.c-ground-nav__legal {
+  align-self: end; // 1
+  font-size: size.$font-small;
```

Comment:
I believe this `font-size` rule can be omitted. Alignment is improved when it's the same size as everything else.

Reply:
```suggestion
```